### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by formulahendry**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "azure-iot-toolkit",
-  "displayName": "Azure IoT Hub Toolkit",
-  "description": "Interact with Azure IoT Hub, IoT Device Management, IoT Edge Management, IoT Hub Device Simulation, IoT Hub Code Generation and IoT Hub Device Provisioning Service (DPS)",
+  "displayName": "Azure IoT Hub",
+  "description": "This extension is now a part of Azure IoT Tools extension pack. We highly recommend installing Azure IoT Tools to get full capabilities for Azure IoT development.",
```
Comment (blackchoey) [inline package.json]: Also keep the previous content in description?

Comment (adashen) [reply package.json]: Agree, we need keep the previous description to let user know what the extension help.

File: src/welcomePage.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export class WelcomePage {
             const startTime = new Date();
             this.panel = vscode.window.createWebviewPanel(
                 "welcomePage",
-                "Welcome to Azure IoT Hub Toolkit",
+                "Welcome to Azure IoT Hub extension",
```
Comment (adashen) [inline src/welcomePage.ts]: Extension?

**Comment by blackchoey** [comment]
Searching `Azure IoT Hub Toolkit`, there're some other places we may need to update. E.g. (not full list)
* Promote message in https://github.com/microsoft/vscode-azure-iot-toolkit/blob/master/src/nsat.ts
* Output channel name in https://github.com/microsoft/vscode-azure-iot-toolkit/blob/master/src/extension.ts

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "azure-iot-toolkit",
   "displayName": "Azure IoT Hub",
-  "description": "This extension is now a part of Azure IoT Tools extension pack. We highly recommend installing Azure IoT Tools to get full capabilities for Azure IoT development.",
+  "description": "Interact with Azure IoT Hub, IoT Device Management, IoT Edge Management, IoT Hub Device Simulation, IoT Hub Code Generation and IoT Hub Device Provisioning Service (DPS). This extension is now a part of Azure IoT Tools extension pack. We highly recommend installing Azure IoT Tools to get full capabilities for Azure IoT development.",
```
Comment (adashen) [inline package.json]: The new description should be moved to the beginning...

Comment (formulahendry) [reply package.json]: updated

**Pull request merged**